src/js/index.tsx

```diff
@@ -92,6 +93,7 @@ const App = withRouter(
               <ErrorBoundary>
                 <Switch>
                   <Route exact path={routePatterns.login} component={Login} />
+                  <Route exact path={routePatterns.manage} component={Manage} />
```

Comment:
We probably want to use `<PrivateRoute>` here so that this page is only available when logged in.

```suggestion
                  <PrivateRoute exact path={routePatterns.manage} component={Manage} />
```